mne/viz/topomap.py

```diff
@@ -996,12 +997,32 @@ def plot_ica_components(ica, picks=None, ch_type=None, res=64,
     fig.subplots_adjust(top=0.95)
     fig.canvas.draw()
     if isinstance(inst, (BaseRaw, BaseEpochs)):
-        def onclick(event, ica=ica, inst=inst):
+        def onclick(event, ica=ica, inst=inst, titles=titles):
```

Comment:
I don't have to add titles here, I don't know if it matters speed-wise (should be negligable).